lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1912,6 +1915,10 @@ def cmd_sessions(*args)
               session.init_ui(driver.input, driver.output)
               session.execute_script('post/multi/manage/shell_to_meterpreter')
               session.reset_ui
+            elsif session.type == 'powershell'
+              session.init_ui(driver.input, driver.output)
+              session.execute_script('post/multi/manage/shell_to_meterpreter')
```

Comment:
Should this be "powershell_to_meterpreter" instead, or does this post module know how to handle PS?